### DISCUSSION TITLE
Using multipart body in both connector & request

### DISCUSSION THREAD

**Discussion by arsel-5lvxa7b1** [saloonphp/saloon#383] [Q&A]
Hello! Seems like having a common parameter in connector & having own parameters in request is not possible now in Saloon (version 3.7.0).

I tried to send a request to the same Laravel server and request has no data with `->all()` or `->getContent()`.

Also in documentation there seems to be an error [here](https://docs.saloon.dev/the-basics/request-body-data/multipart-form-body#connector-body), there is no `HasMultipartValue` trait, I tried to use `HasMultipartBody`, doesn't work.

Maybe it's because of different boundaries in connector & request?

Also, when using Laravel HTTP sender, the parameters are not saved by Telescope on HTTP Client page. When I use `Http::asMultipart()->post()`, I can see the parameters.

**Comment by Sammyjo20** [comment]
Hey @ArselMuginov thank you for your patience on my response. 

I'm a little confused by your issue, unfortunately - merging multipart bodies should work as only one boundary is picked (I believe). Are the properties called the same name or is each multipart value different?

Thanks
Sam

**Reply by arsel-5lvxa7b1** [reply]
Hello, the names are different, also I only use plain values (name & value without files).

I was debugging it when it wasn't working, dd the pending request body and it had all the values I added, both from connector & request, although I don't remember if boundaries were different.
